### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Yuriy-Komolov**
[DEMO LINK](https://Yuriy-Komolov.github.io/react_person/)

### CODE REVIEW

**Review by Oleksandra-Shevchenko — APPROVED** [review]
LGTM 🥇

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,39 @@
...
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <>
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:18]: useless

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,39 @@
...
+      <section className="Person">
+        <h2 className="Person__name">
+          {`My name is ${name}`}
+        </h2>
+
+        {age
+          ? <p className="Person__age">{`I am ${age}`}</p>
+          : null
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:26]: use && operator

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,39 @@
...
+        }
+
+        <p className="Person__partner">
+          {isMarried ? (
+            `${partnerName} is my ${partner}`
+          ) : (
+            'I am not married'
+          )}
```
Comment (Andr1yK) [inline src/components/Person/Person.jsx:34]:
```suggestion
          {isMarried
            ? `${partnerName} is my ${partner}`
            :  'I am not married'
          }
```

**Review by Andr1yK — CHANGES_REQUESTED** [review]

**Pull request closed without merging**